### ISSUE TITLE
composer.json: silverstripe-australia is symbiote now!!

### ISSUE THREAD

**Issue by RoyalPulp** [dnadesign/silverstripe-elemental#101]
These 2 lines in composer.json should be changed:
```
"silverstripe-australia/gridfieldextensions": "~1.0",
"silverstripe-australia/addressable": "~1.0",
```

to:
```
"symbiote/silverstripe-gridfieldextensions": "~1.0",
"symbiote/silverstripe-addressable": "~1.0",
```

**Comment by robbieaverill** [comment]
@RoyalPulp would you like to make a pull request to update this?

**Comment by wilr** [comment]
Fixed in 11f90f905be77285a56b0cd78c9a7f3912b76528
